Build is failing with parse error with `ENABLE_MOUSE_KEYS` defined

I managed to build the following layout a few days ago:

https://my.glove80.com/#/layout/user/96f488ed-4f88-464e-b6c6-0918380c49b0

Now it is failing. I managed to figure out, that it has to do with the `ENABLE_MOUSE_KEYS` define.

This is the resulting error:
```
Loading Zephyr default modules (Zephyr base (cached)).
-- Application: /nix/store/kmfgl81pcqd2hj9ingbchmna6ilx1cb6-source/app
-- CMake version: 3.30.5
-- Found Python3: /nix/store/3ym73yjdw7yg6sv3nwr0syligji76i3z-python3-3.12.7-env/bin/python (found suitable version "3.12.7", minimum required is "3.8") found components: Interpreter
-- Cache files will be written to: /tmp/.cache
-- Zephyr version: 3.5.0 (/nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr), build: 598b03508c308de2139c61216eb1b2fdc447501e
-- Using keymap file: /tmp/build/build.keymap
-- Board: glove80_lh
-- Found toolchain: gnuarmemb (/nix/store/8c42z0y476v6dg16v2phaxjg6kj5m2xr-ccache-links)
-- Found Dtc: /nix/store/2nvmq98zj69i1af1ksh6m08hvfdkjv1z-dtc-1.7.2/bin/dtc (found suitable version "1.7.2", minimum required is "1.4.6")
-- Found BOARD.dts: /nix/store/kmfgl81pcqd2hj9ingbchmna6ilx1cb6-source/app/boards/arm/glove80/glove80_lh.dts
-- Found devicetree overlay: /tmp/build/build.keymap
devicetree error: Label 'zip_scroll_scaler' appears on /input_processors/zip_scroll_scaler and on /zip_scroll_scaler
-- In: /tmp/build/zephyr, command: /nix/store/3ym73yjdw7yg6sv3nwr0syligji76i3z-python3-3.12.7-env/bin/python;/nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/scripts/dts/gen_defines.py;--dts;/tmp/build/zephyr/zephyr.dts.pre;--dtc-flags;'-Wno-unique_unit_address_if_enabled';--bindings-dirs;/nix/store/kmfgl81pcqd2hj9ingbchmna6ilx1cb6-source/app/module/dts/bindings;/nix/store/kmfgl81pcqd2hj9ingbchmna6ilx1cb6-source/app/dts/bindings;/nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/dts/bindings;--header-out;/tmp/build/zephyr/include/generated/devicetree_generated.h.new;--dts-out;/tmp/build/zephyr/zephyr.dts.new;--edt-pickle-out;/tmp/build/zephyr/edt.pickle;--vendor-prefixes;/nix/store/kmfgl81pcqd2hj9ingbchmna6ilx1cb6-source/app/dts/bindings/vendor-prefixes.txt;--vendor-prefixes;/nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/dts/bindings/vendor-prefixes.txt
CMake Error at /nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/cmake/modules/dts.cmake:279 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  /nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/cmake/modules/zephyr_default.cmake:129 (include)
  /nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /nix/store/kv2vw4bhcf23h2zjs7h8sxky9jsbzr1p-zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:97 (include_boilerplate)
  CMakeLists.txt:9 (find_package)


-- Configuring incomplete, errors occurred!
Error time: Fri, 16 May 2025 22:22:42 GMT
```

--- a/keymap.dtsi
+++ b/keymap.dtsi
@@ -9915,7 +9915,7 @@ UNICODE_MORPH_MODS
   #include <zephyr/dt-bindings/input/input-event-codes.h>
   / {
     input_processors {
-      zip_scroll_scaler: zip_scroll_scaler {
+      zip_scroll_scaling_processor: zip_scroll_scaling_processor {
         compatible = "zmk,input-processor-scaler";
         #input-processor-cells = <2>;
         type = <INPUT_EV_REL>;
@@ -9928,19 +9928,19 @@ UNICODE_MORPH_MODS
     #ifdef LAYER_MouseSlow
     slow {
       layers = <LAYER_MouseSlow>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_SLOW>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_SLOW>;
     };
     #endif
     #ifdef LAYER_MouseFast
     fast {
       layers = <LAYER_MouseFast>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_FAST>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_FAST>;
     };
     #endif
     #ifdef LAYER_MouseWarp
     warp {
       layers = <LAYER_MouseWarp>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_WARP>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_WARP>;
     };
     #endif
   };

--- a/keymap.dtsi.erb
+++ b/keymap.dtsi.erb
@@ -2005,7 +2005,7 @@ macros {
   #include <zephyr/dt-bindings/input/input-event-codes.h>
   / {
     input_processors {
-      zip_scroll_scaler: zip_scroll_scaler {
+      zip_scroll_scaling_processor: zip_scroll_scaling_processor {
         compatible = "zmk,input-processor-scaler";
         #input-processor-cells = <2>;
         type = <INPUT_EV_REL>;
@@ -2018,19 +2018,19 @@ macros {
     #ifdef LAYER_MouseSlow
     slow {
       layers = <LAYER_MouseSlow>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_SLOW>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_SLOW>;
     };
     #endif
     #ifdef LAYER_MouseFast
     fast {
       layers = <LAYER_MouseFast>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_FAST>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_FAST>;
     };
     #endif
     #ifdef LAYER_MouseWarp
     warp {
       layers = <LAYER_MouseWarp>;
-      input-processors = <&zip_scroll_scaler MOUSE_SCALER_WARP>;
+      input-processors = <&zip_scroll_scaling_processor MOUSE_SCALER_WARP>;
     };
     #endif
   };

